README.md

```diff
@@ -1,2 +1,3 @@
-# devrel
-Anything Developer Relations at Meili
+# Developer Relations at MeiliSearch
+
+We will slowly add all our processes in this repo. This is how we rool at MeiliSearch, open source to the core! You can also have an idea of what we are working on by checking the [issues](https://github.com/meilisearch/devrel/issues) (for devrel tasks only).
```

Comment:
```suggestion
We will slowly add all our processes in this repo. This is how we rule at MeiliSearch, open source to the core! You can also have an idea of what we are working on by checking the [issues](https://github.com/meilisearch/devrel/issues) (for devrel tasks only).
```

You mean 'rule', right?

Reply:
I think he meant `roll` 😄

Reply:
https://www.ldoceonline.com/dictionary/that-s-how-i-roll-it-s-how-i-roll
In case you didn't know this phrase.

Reply:
Yeah, I meant roll!

Reply:
I'll fix it